configs/fedramp-prod/roles/inventory.json

```diff
@@ -5,6 +5,7 @@
       "description": "Perform any available operation against any Inventory resource.",
       "system": true,
       "platform_default": true,
+      "admin_default": true,
```

Comment:
we don't need to change files fedram-* directories.

Reply:
I've heard that there are plans for fedramp environments to come back, that's why also made the change here. But if this is an issue, I don't have a problem with removing this change.

Reply:
yes maybe you can remove it and they will add it if it will be needed, thanks!

Reply:
Is this comment still valid? Should I remove roles/permissions from fedramp, or should I keep it in the PR?

Reply:
Done